libs/client-api/src/http.rs

```diff
@@ -844,9 +847,12 @@ impl Client {
     &self,
     method: Method,
     url: &str,
+    serialization_type: &SerializationType,
   ) -> Result<RequestBuilder, AppResponseError> {
     #[cfg(feature = "enable_brotli")]
     {
+      let serialization_type_header_value =
+        HeaderValue::from_str(serialization_type.to_string().as_str()).unwrap();
```

Comment:
is it safe to unwrap?

Reply:
The only scenario where this will panic is if we wrongly include non-ASCII characters like `\n` in the to string method for Serialization Type enum. Hence, this should be safe to unwrap.

Reply:
Understood. It would be better to make a annotation for this case. For example: //safety: xxx